### ISSUE TITLE
assertion fail when creating new record

### ISSUE THREAD

**Issue by BryanHunt** [danielspaniel/ember-data-change-tracker#30]
EmberJS + EmberData 2.15

When you create a new record, you get the following assertion failure.  I tried setting the changeTracker options both on the model, and in the environment.

```
ember-metal.js:3988 Error: Assertion Failed: You need to pass a model name to the store's serializerFor method
    at new EmberError (error.js:38)
    at Object.assert (index.js:142)
    at Class.serializerFor (-private.js:12472)
    at Function.transformFn (tracker.js:137)
    at Function.getTransform (tracker.js:261)
    at tracker.js:220
    at Array.forEach (<anonymous>)
    at Function.getTrackerInfo (tracker.js:217)
    at Function.setupTracking (tracker.js:183)
    at Class.setupTrackerMetaData (model-ext.js:47)
```

**Comment by danielspaniel** [comment]
Can you snoop around and see if you can fix this one or find a solution?  I am out of commission for the next few days, so can't get to it for a while.

**Comment by danielspaniel** [comment]
Actually .. can you submit a PR with a test showing this failure? That helps me to see the problem and then find a solution faster.

**Comment by BryanHunt** [comment]
If I'm not gork'd out from my knee surgery tomorrow, I'll try to put something together this weekend.

**Comment by ktrhn** [comment]
The error seems to happen with custom defined transforms (https://www.emberjs.com/api/ember-data/2.14/classes/DS.Transform)
Problem is that `model.constructor` does not seem to be available anymore. Changing to `model._internalModel` helps, but uses private api... There may be a better solution, I am not so much into the ember-data internals. PR is incoming.

**Comment by danielspaniel** [comment]
This was such a goofy bug. It is now fixed and tested => check out v0.7.1
Thank you for the bug report.
